lib/assessment.js

```diff
@@ -8,15 +8,17 @@
 // Question 1
 // Write a callback for reduce that selects
 // the minimum value in the array
-var min = function min() {
-
+var min = function min(p, c, i, a) {
+  var result = p;
+  if(result > c) result = c;
+  return result;
```

Comment:
Yes, this will work (though I'm not sure if you might need curly braces after the `if`). 

Here's another way that it could have been written.
```javascript
var min = function min (prev, curr, index, arr) {
  return (prev < curr)? prev : curr;
};
```